torch/testing/_internal/opinfo/definitions/linalg.py

```diff
@@ -2399,6 +2399,12 @@ def make_input():
     #
     # torch.linalg
     #
+    PythonRefInfo(
+        "_refs.linalg.cross",
+        torch_opinfo_name="linalg.cross",
+        supports_out=True,
+        op_db=op_db
+    ),
```

Comment:
I think just adding a skip is fine here.

Reply:
I added the xfails above and they seem to have squashed most of the problems.

The second one does not sit right with me though. The issue is that when the test executes the op [here](https://github.com/pytorch/pytorch/blob/0e6eee3c898c293efc3f172180e9f4d79cc0b13f/test/test_ops.py#L576). The arguments (a, b) are `FakeTensor` instances and the ref implementation creates the `idx` tensor from non-tensor operands. 

The test does not execute with the `FakeTensorMode()` context active, but the failure indicates that it is expected that `idx` will be a `FakeTensor`. Given that a factory is used and there are no `Tensor` arguments, how is that supposed to be possible? Is there a solution I have missed or is the xfail here correct?

Reply:
Ag, yeah, I guess we never hit this issue because we don't do any advanced indexing in refs. I guess the xfail + a comment is fine. Just make sure the error checks match those from eager.